R/pull.R

```diff
@@ -22,7 +22,13 @@ pulley <- function(resamples, res, col) {
   all_null <- all(purrr::map_lgl(res, is.null))
 
   id_cols <- grep("^id", names(resamples), value = TRUE)
-  resamples <- dplyr::arrange(resamples, !!!syms(id_cols))
+
+  if (is.null(order)) {
+    order <- vctrs::vec_order(resamples[id_cols])
+  }
+
```

Comment:
is this needed since you always pass in the order when `pulley()` is used?

Reply:
Thank you! Heard—[81749cc](https://github.com/tidymodels/tune/pull/730/commits/81749cc9a25c12c8b0970ec85531f1fb25872ab2).